espnet2/asr/espnet_model.py

```diff
@@ -558,3 +558,36 @@ def _calc_transducer_loss(
             )
 
         return loss_transducer, cer_transducer, wer_transducer
+
+    def _calc_batch_ctc_loss(
```

Comment:
It this function included in the unit test?

Reply:
Yes, it is implicitly covered by `test_enh_asr_model_2spk` in `test/espnet2/enh/test_espnet_enh_s2t_model.py` when `calc_enh_loss` is True.